packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/SimulationResource.java

```diff
@@ -169,7 +170,7 @@ public Publisher<byte[]> stream(
 	) {
 		ObjectMapper mapper = new ObjectMapper();
 		return Multi.createFrom().publisher(simulationStatusStream).filter(event -> {
-			try{ 
+			try{
```

Comment:
You should also add a try catch to your endpoint

Reply:
💯

Reply:
Done!